setup_sklearnex.py

```diff
@@ -13,17 +13,27 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-#===============================================================================
+# ===============================================================================
 
 # System imports
 import os
 import time
 from setuptools import setup
+from scripts.version import get_onedal_version
 from scripts.package_helpers import get_packages_with_tests
 
 sklearnex_version = (os.environ["SKLEARNEX_VERSION"] if "SKLEARNEX_VERSION" in os.environ
                      else time.strftime("%Y%m%d.%H%M%S"))
 
+dal_root = os.environ.get('DALROOT')
+
+if dal_root is None:
+    raise RuntimeError("Not set DALROOT variable")
+
+ONEDAL_MAJOR_BINARY_VERSION, ONEDAL_MINOR_BINARY_VERSION = get_onedal_version(
+    dal_root, 'binary')
```

Comment:
Binary versions are not used in this script

Reply:
Fixed